pkg/controller/job-controller.go

```diff
@@ -85,7 +79,28 @@ func (c *JobController) processNextJobWorkItem() bool {
 		}
 		klog.V(2).Infof("Key from workqueue: %s", key)
 
-		c.SyncHandler(key)
+		// Run the syncHandler, passing it the namespace/name string of the
+		result, err := c.SyncHandler(key)
+		switch {
+		case err != nil:
+			c.workqueue.AddRateLimited(key)
+			return fmt.Errorf("error syncing '%s': %s", key, err.Error())
+		case result.RequeueAfter > 0:
+			// The result.RequeueAfter request will be lost, if it is returned
+			// along with a non-nil error. But this is intended as
+			// We need to drive to stable reconcile loops before queuing due
+			// to result.RequestAfter
+			c.workqueue.Forget(obj)
+			c.workqueue.AddAfter(key, result.RequeueAfter)
+		case result.Requeue:
+			c.workqueue.AddRateLimited(key)
+		default:
+			// Finally, if no error occurs we Forget this item so it does not
+			// get queued again until another change happens.
+			c.workqueue.Forget(obj)
+			klog.V(4).Infof("Successfully synced '%s'", key)
```

Comment:
same here, I think we should keep verbosity minimal during development of the MinJob project please